types/ical/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for ical 0.6
+// Type definitions for ical 0.8
 // Project: https://github.com/peterbraden/ical.js
 // Definitions by: Nick Clifford <https://github.com/nickbclifford>
 //                 Matej Vilk <https://github.com/iammatis>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// Minimum TypeScript Version: 3.9
 
 import request = require('request');
```

Comment:
If these types are no longer required, let's remove this import.

Reply:
Good catch! Thanks, removed.